TangoRosStreamer/tango_ros_node/src/main/java/eu/intermodalics/tango_ros_node/TangoRosNode.java

```diff
@@ -64,6 +64,8 @@ public TangoRosNode(String libName) {
     @Override
     public native int shutdown();
 
+    public native boolean saveMap(String mapName);
```

Comment:
Good point. My initial thought was that we would have both: this new API and a ROS service (not yet implemented), but that might be redundant.